bin/ch/WScriptJsrt.cpp

```diff
@@ -511,6 +554,114 @@ JsValueRef WScriptJsrt::LoadScript(JsValueRef callee, LPCSTR fileName,
     return value;
 }
 
+ThreadData::ThreadData()
```

Comment:
sure. for naming this is intended to generally support 'crossthread', not only for $262.agent. I'm going with RuntimeThreadData